sql_generators/glean_usage/templates/cross_channel_events_stream.query.sql

```diff
@@ -16,5 +16,5 @@ SELECT
     "{{ channel }}" AS normalized_channel
     {% endif -%}
   ),
-FROM `{{ project_id }}.{{ dataset }}_derived.events_stream` AS e
+FROM `{{ project_id }}.{{ dataset }}.events_stream` AS e
```

Comment:
this should get us `fenix` views

Reply:
does this mean the output datasets will change locations?

Reply:
@scholtzan what wil asks. And how can we verify this?

Reply:
Nothing is changing locations from this change. It's just that we either need to reference the views (like `org_mozilla_fenix.events_stream`) or we could also reference the tables which are in the `_derived` datasets and add the table version (`org_mozilla_fenix_derived.events_stream_v1`)